modules/installation-initializing-manual.adoc

```diff
@@ -17,7 +17,7 @@ endif::[]
 = Manually creating the installation configuration file
 
 For installations of {product-title} that use user-provisioned
-infrastructure, you must manually generate your installation configuration file.
+infrastructure, you manually generate your installation configuration file.
```

Comment:
Since installer is not able to access the dns zone in host project, I suggest to demonstrate that **with shared VPC**, you must manually generate your installation configuration file.